record/record.app.js

```diff
@@ -144,6 +144,9 @@
                     DataUtils.getDisplaynameInnerText(tuple.displayname)
                 );
 
+                // populate the google dataset metadata
+                recordAppUtils.attachGoogleDatasetJsonLd(tuple);
```

Comment:
Let's move this to be after line that says `$window.history.replaceState({}, '', url);`. If this code is going to use the window.location, we have to make sure it's picking up the correct value.